packages/qdrant-loader/src/qdrant_loader/core/pipeline/source_processor.py

```
@@ -98,3 +107,64 @@ async def process_source_type(
+                        f"Setting file conversion config for {source_type} source: {source_name}"
+                    )
+                    connector.set_file_conversion_config(self.file_conversion_config)
+
+                async with connector:
+                    document_count = 0
+                    async for document in connector.stream_documents(since=since):
+                        yield document
+                        document_count += 1
+                        if document_count % 100 == 0:
+                            logger.debug(
+                                f"Streamed {document_count} documents from {source_type} source: {source_name}"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)